modules/network/smb2man/iopheap.c

```
@@ -0,0 +1,114 @@
+
+#define ALLOC_HDR_SIZE 16
+#define ALLOC_MAGIC    0x5342324D // "SB2M"
+
+struct alloc_hdr
+{
+    u32 magic;
+    u32 size; // payload size, not including this header
+    u32 pad[2];
+};
+
+void *malloc(size_t size)
```

Cppcheck (2.21.0)

[style] The function 'malloc' should have static linkage since it is not used outside of its translation unit.

(staticFunction)